src/main/java/org/killbill/billing/entitlement/api/Entitlement.java

```diff
@@ -203,6 +203,28 @@ public enum EntitlementSourceType {
     public Entitlement cancelEntitlementWithDate(final LocalDate effectiveDate, final boolean overrideBillingEffectiveDate, final Iterable<PluginProperty> properties, final CallContext context)
             throws EntitlementApiException;
 
+    
+    /**
+     * Cancels the <code>Entitlement</code> with the specified entitlementEffectiveDate and billingEffectiveDate
+     * After this operation, the existing object becomes stale.
+     * <p/>
+     * <p/>
+     *
+     * @param entitlementEffectiveDate                the datetime at which the entitlement should be cancelled
```

Comment:
Nit: vertical alignment of the Javadocs. See above, although the alignment is a bit whacky there as well! One day, we should do a proper pass at cleaning up and reviewing all of theses Javadocs.